src/redux/reducers/componentConfig/initialState.js

```diff
@@ -1,90 +1,126 @@
+import {
+  legendBaseState,
+  dimensionsBaseState,
+  axesBaseState,
+  titleBaseState,
+  fontStyleBaseState,
+  colourBaseState,
+  markerBaseState,
+  labelBaseState,
+} from './baseStylesState';
+
+// PLOTS & TABLES - Categorical Embedding
 const embeddingCategoricalInitialConfig = {
   spec: '1.0.0',
-  width: 700,
-  height: 550,
-  pointSize: 5,
-  toggleInvert: '#FFFFFF',
-  masterColour: '#000000',
-  titleText: '',
-  titleSize: 20,
-  titleAnchor: 'start',
-  axisTitlesize: 13,
-  axisTicks: 13,
-  transGrid: 0,
-  axesOffset: 10,
-  masterFont: 'sans-serif',
-  xaxisText: 'UMAP 1',
-  yaxisText: 'UMAP 2',
-  pointStyle: 'circle',
-  pointOpa: 5,
-  g1Color: 'red',
-  g2mColor: 'green',
-  sColor: 'blue',
-  legendTextColor: '#FFFFFF',
-  geneexpLegendloc: '',
-  labelSize: 28,
-  labelShow: 1,
-  labelFont: 2,
-  labelsEnabled: true,
+  legend: {
+    ...legendBaseState,
+    position: 'top',
+  },
+  dimensions: {
+    ...dimensionsBaseState,
+    width: 700,
+    height: 550,
+  },
+  axes: {
+    ...axesBaseState,
+    xAxisText: 'UMAP 1',
+    yAxisText: 'UMAP 2',
+    offset: 10,
+  },
+  title: {
+    ...titleBaseState,
+    fontSize: 20,
+  },
+  fontStyle: fontStyleBaseState,
+  colour: colourBaseState,
+  marker: markerBaseState,
+  label: labelBaseState,
   selectedCellSet: 'louvain',
-  testVar: null,
-  legendEnabled: true,
-  legendPosition: 'top',
+  selectedSample: 'All',
 };
 
+// PLOTS & TABLES - Continuous Embedding
 const embeddingContinuousInitialConfig = {
   spec: '1.0.0',
-  width: 700,
-  height: 550,
-  selectedSample: 'All',
-  pointSize: 5,
-  colGradient: 'viridis',
-  toggleInvert: '#FFFFFF',
-  masterColour: '#000000',
-  reverseCbar: false,
+  legend: legendBaseState,
+  dimensions: {
+    ...dimensionsBaseState,
+    width: 700,
+    height: 550,
+  },
+  axes: {
+    ...axesBaseState,
+    xAxisText: 'UMAP 1',
+    yAxisText: 'UMAP 2',
+    domainWidth: 2,
+    offset: 10,
+  },
+  title: {
+    ...titleBaseState,
+    dx: 0,
+    fontSize: 20,
+  },
+  fontStyle: fontStyleBaseState,
+  colour: colourBaseState,
+  marker: markerBaseState,
+  label: labelBaseState,
   logEquation: 'datum.expression*1',
-  titleText: '',
-  shownGene: 'notSelected',
-  titleSize: 20,
-  titleAnchor: 'start',
-  axisTitlesize: 13,
-  axisTicks: 13,
-  transGrid: 0,
-  axesOffset: 10,
-  masterFont: 'sans-serif',
-  xaxisText: 'UMAP 1',
-  yaxisText: 'UMAP 2',
-  pointStyle: 'circle',
-  pointOpa: 5,
-  bounceX: 0,
-  legendEnabled: true,
-  legendPosition: 'top-right',
+  shownGene: 'CST3',
```

Comment:
this should be changed back to `notSelected`, or the continuous embedding will error for datasets that don't have gene `CST3` (fixed as part of: https://github.com/biomage-ltd/ui/pull/92)

Reply:
ok, will remove in next commit, together with bug fixes.